memecity.game/Game/Systems/InputSystem.cpp

```diff
@@ -29,19 +29,19 @@ void InputSystem::run(EntityManager& em) const
 		auto velocity_component = entity.get<VelocityComponent>();
 		if (input_manager.is_down(input::UP))
 		{
-			velocity_component->y -= 0.1f;
+			velocity_component->y = -1.1f;
 		}
 		if (input_manager.is_down(input::DOWN))
 		{
-			velocity_component->y += 0.1f;
+			velocity_component->y += 1.1f;
 		}
 		if (input_manager.is_down(input::LEFT))
 		{
-			velocity_component->x -= 0.1f;
+			velocity_component->x = -1.1f;
```

Comment:
assignment ipv modification? waar het bij de rest wel -= of += is is het hier een =